spring-cloud-gcp-pubsub-stream-binder/src/test/java/com/google/cloud/spring/stream/binder/pubsub/config/PubSubBinderConfigurationTests.java

```diff
@@ -165,7 +167,10 @@ void test_healthRegistryEnabled() {
   private static class TestConfiguration {
     @Bean
     public CredentialsProvider googleCredentials() {
-      return () -> mock(Credentials.class);
+      Credentials mockCredential = mock(Credentials.class);
+      when(mockCredential.getMetricsCredentialType())
+          .thenReturn(CredentialTypeForMetrics.DO_NOT_SEND);
```

Comment:
If mock credentials were placed in core, could you avoid the duplication here?

Reply:
The mock credentials are placed under "test" folder, not accessible here.
`spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/TestUtils.java`
Created another util class in this module to avoid duplicating logic.